guides/plugins/plugins/administration/adding-responsive-behavior.md

```diff
@@ -0,0 +1,52 @@
+# Adding responsive behavior
+
+## Overview
+The Shopware 6 Administration provides two ways of adding classes to elements based on their size, the device helper and the `v-responsive` directive.
+Alternatively you can use `sccs` media queries to make your plugin responsive.
```

Comment:
`sccs` -> `scss`

How are scss media queries different css media queries?

Reply:
There is no difference at all. I'd suggest you refer to `CSS` queries here